2023-03-10-3mdeb-msi-pro-z690-a-ddr4-first-qubes-certified-desktop.md

```diff
@@ -0,0 +1,56 @@
+| CPU	         | Intel Core i5-12600K, 3.7GHz                                    |
+| Cooling	     | Noctua CPU NH-U12S Redux (w/ Noctua NM-i17xx-MP78 Mounting Kit) |
+| RAM	         | Kingston Fury Beast, DDR4, 4x8GB (32 GB Total), 3600 MHz, CL17  |
+| Power Supply | Seasonic Focus PX 750W 80 Plus Platinum                         |
+| Storage      | SSD Intel 670p 512 GB M.2 2280 PCI-E x4 Gen3 NVMe               |
+| Enclosure	   | SilentiumPC Armis AR1                                           |
+
+[![Photo of 3mdeb MSI PRO Z690-A DDR4 with open case](/attachment/posts/3mdeb-msi-pro-z690-a-ddr4_2.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/)
+
+As 3mdeb explains, "The integral part of this product is [Dasharo Tools Suite Supporters Entrance](https://docs.dasharo.com/osf-trivia-list/dts/#what-is-dasharo-tools-suite-supporters-entrance)," or "DTS SE," which provides access to Dasharo firmware updates for your hardware. This includes:
+
+- The latest Dasharo release installed by the Dasharo Team
+- Dasharo updates, where the number of updates depends on the number of Dasharo subscriptions sold and the availability of other funding (e.g., NLNet, corporate sponsors, community donations)
+- Priority support through an invite-only Matrix channel
+- Influence on the Dasharo feature roadmap
+
+3mdeb points out that, by purchasing the MSI PRO Z690-A DDR4 with DST SE, you can have a meaningful impact on the course of Dasharo's development, as well as support the development of open-source firmware and the Dasharo distribution itself.
+
+[![Photo of 3mdeb MSI PRO Z690-A DDR4 with open case](/attachment/posts/3mdeb-msi-pro-z690-a-ddr4_3.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/)
+
+For further details, please see the [MSI PRO Z690-A DDR4](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/) product page.
+
+[![Photo of the outside of the 3mdeb MSI PRO Z690-A DDR4](/attachment/posts/3mdeb-msi-pro-z690-a-ddr4_4.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-compatible-with-msi-pro-z-690a-ddr4-full-pc-build/)
+
+## Special note regarding the need for `kernel-latest`
+
+Beginning with Qubes OS 4.1.2, the Qubes installer includes the `kernel-latest` package and allows users to select this kernel option when installing. At the time of this announcement, `kernel-latest` is **required** for the MSI PRO Z690-A DDR4's graphics drivers to function properly. Therefore, all potential purchasers and users of this model should be aware that they will have to select a non-default option (`kernel-latest`) during installation. However, since Linux 6.1 has officially been promoted to being a long-term support (LTS) kernel, it will become the default kernel at some point, which means that the need for this non-default selection is only temporary.
```

Comment:
Maybe I have missed it during the installation or the `kernel-latest` option was not visible. I had to install it manually after launching Qubes OS by using `qubes-dom--update`.

![image](https://user-images.githubusercontent.com/30656491/223096872-3f5e4c62-aa67-473b-bf01-151c8cce8fb0.png)

@marmarek can you confirm the kernel-latest option is available at the installer's stage?

Reply:
Yes, that's only in 4.1.2 builds, earlier builds didn't have it. I'm talking about earlier selection during installer startup: https://openqa.qubes-os.org/tests/68192#step/install_startup/1

Reply:
Ahh it is in the GRUB menu... Okay. Thanks.

Reply:
Clarified this in the description: https://github.com/QubesOS/qubes-posts/pull/108/commits/7958f6f5e5f5e39a243042751e23d6234465b982